src/Test/Perf/Runner/Perf.Runner.csproj

```diff
@@ -80,4 +80,10 @@
     <Folder Include="CPC-Consumption\" />
   </ItemGroup>
   <Import Project="..\..\..\..\build\Targets\VSL.Imports.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>call "$(MSBuildThisFileDirectory)install_benchview_tools.cmd" "$(OutDir)"</PostBuildEvent>
```

Comment:
I moved the acquisition of the benchview tools out of Perf.Runner.csproj. They are now acquired only when they will be utilized.